.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     - uses: actions/checkout@v2
     - name: Run linters
       run: ./automation/lint.sh
-  test-storage:
+  test-storage-user:
```

Comment:
Please keep test-storage for now, since github setting is configured to require this build, and we don't have permissions to change it.

Add a comment to rename it once we change github settings to require test-storage-user and test-storage-root.